VRDR/DeathRecord.cs

```diff
@@ -881,6 +882,38 @@ public Bundle GetDemographicCodedContentBundle()
             AddResourceToBundleIfPresent(InputRaceAndEthnicityObs, dccBundle);
             return dccBundle;
         }
+        /// <summary>Helper method to return the subset of this record that makes up a Mortality Roster bundle.</summary>
+        /// <returns>a new FHIR Bundle</returns>
+        public Bundle GetMortalityRosterBundle(Boolean alias)
+        {
+            Bundle mortRosterBundle = new Bundle();
+            mortRosterBundle.Id = Guid.NewGuid().ToString();
+            mortRosterBundle.Type = Bundle.BundleType.Collection;
+            mortRosterBundle.Meta = new Meta();
+            string[] profile = { ProfileURL.MortalityRosterBundle };
+            mortRosterBundle.Meta.Profile = profile;
+            mortRosterBundle.Timestamp = DateTime.Now;
+            // Make sure to include the base identifiers, including certificate number and auxiliary state IDs
```

Comment:
this comment sounds like this hasn't been confirmed yet or is a TODO, but line 897 should set all of those id's right? update the comment to say that it does include the base identifiers